src/pages/NewSensor/NewSensorStepTwo.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { TabContainer } from './TabContainer';
+import { BreachConfigRow } from './BreachConfigRow';
+import { PaperSection, FlexRow, PageButton, Spacer } from '../../widgets';
+
+import { SUSSOL_ORANGE, WHITE } from '../../globalStyles';
+import { vaccineStrings } from '../../localization';
+import { WizardActions } from '../../actions/WizardActions';
+import { goBack } from '../../navigation/actions';
+
+export const NewSensorStepTwoComponent = ({ nextTab, previousTab, exit }) => (
+  <TabContainer>
+    <PaperSection height={320} headerText={vaccineStrings.new_sensor_step_two_title}>
```

Comment:
the fixed values for heights are necessary?

Reply:
If I say yes do I have to explain more..? Because I just know that no look good without 😅

Reply:
ok, I'm convinced. that's a strong argument you've put forward there.